cloudbuild.yaml

```diff
@@ -43,19 +43,19 @@ steps:
     waitFor: ['build_image']
     timeout: 30m
     entrypoint: 'bash'
-    args: ['./scripts/build_server_asylo']
+    args: ['./scripts/build_server', '-s', 'asylo']
```

Comment:
Could you confirm that after running `build_server` with each `-s` value once, and then re-running all of them again, the second time all of the builds are cached and finish immediately? And also for `build_example`.

Reply:
Confirming
Also CI build still takes 20 minutes
https://pantheon.corp.google.com/cloud-build/builds/ce14acc6-f8e2-4900-9756-8804fe028dac?project=oak-ci

[build_cached.txt](https://github.com/project-oak/oak/files/4268299/build_cached.txt)